### PR TITLE
drag drop initial build

### PR DESCRIPTION
**Pull request by Bharat-Asawale**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# this are the ignored files 
+node_modules
+check
```
Comment (girishI9) [inline .gitignore]: add `dist` folder, `package-lock.json` as well

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            <div class="form-control">
+                <label for="people">people</label>
+                <input type="number" id="people" step="1" min="0" max="10"/>
+            </div>
+            <button type="submit">ADD Project</button>
+        </form>
+    </template>
+    <template id="single-project">
+        <li draggable="true">
+            <h2></h2>
```
Comment (girishI9) [inline index.html:31]: Why it is empty?

File: src/app.ts [hunk 3]
```diff
@@ -0,0 +1,376 @@
+// Drag & Drop Interfaces
+interface Draggable{
```
Comment (girishI9) [inline src/app.ts]: Try to create separate file for interface, helper const functions and class

**Review by girishI9 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Bharat-Asawale) [reply index.html:31]: Here the contents are coming from typescript  file

File: src/app.ts (hunk 3 above)
Comment (Bharat-Asawale) [reply src/app.ts]: folder structure is managed

**Pull request merged**